components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/util/AccountUtil.java

```diff
@@ -145,4 +149,40 @@ public static boolean isTemplateExists(String templateType, String tenantDomain)
                             + "tenantDomain: " + tenantDomain, e);
         }
     }
+
+    /**
+     * Publishes an event.
+     *
+     * @param eventName                 Event name
+     * @param properties                Event properties
+     * @throws AccountLockException     if event sent failed
```

Comment:
Shall we use sentence case in all of the parameter/exception descriptions?

Reply:
Fixed in 0203a22b19ea00df2e1ed6c9ad4f61421349e6eb